src/sys/select.rs

```diff
@@ -59,14 +61,31 @@ impl FdSet {
     ///
     /// [`select`]: fn.select.html
     pub fn highest(&mut self) -> Option<RawFd> {
-        for i in (0..FD_SETSIZE).rev() {
-            let i = i as RawFd;
-            if unsafe { libc::FD_ISSET(i, self as *mut _ as *mut libc::fd_set) } {
-                return Some(i)
-            }
-        }
+        self.fds(None).next_back()
+    }
 
-        None
+    /// Returns an iterator over the file descriptors in the set. For
```

Comment:
Put a full blank line after the first full sentence.  That will make the docs on docs.rs formatted nicely.

Reply:
Done.